### ISSUE TITLE
Unhandled exception when installing 2.6-beta.3 on Linux

### ISSUE THREAD

**Issue by Kevin-Mok** [Pathoschild/SMAPI#485]
This came up while installing it on Mint but not on Ubuntu. Used `./install\ on\ Linux.sh` in the unzipped directory with `urxvt` and `zsh`.
```
Unhandled Exception:
System.IO.FileNotFoundException: Could not load file or assembly 'System.Management, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.
File name: 'System.Management, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
  at StardewModdingApi.Installer.InteractiveInstaller.Run (System.String[] args) [0x00025] in <7324f88dc4524a57b82d853323e7f820>:0 
  at StardewModdingApi.Installer.Program.Main (System.String[] args) [0x00005] in <7324f88dc4524a57b82d853323e7f820>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.IO.FileNotFoundException: Could not load file or assembly 'System.Management, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.
File name: 'System.Management, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
  at StardewModdingApi.Installer.InteractiveInstaller.Run (System.String[] args) [0x00025] in <7324f88dc4524a57b82d853323e7f820>:0 
  at StardewModdingApi.Installer.Program.Main (System.String[] args) [0x00005] in <7324f88dc4524a57b82d853323e7f820>:0 
```

**Comment by Pathoschild** [comment]
@Kevin-Mok Can you try reinstalling Mono? The installer works fine for me on ZorinOS, so might be something wrong with Mono on your Mint machine.

**Comment by Kevin-Mok** [comment]
Works fine after `apt update && apt install mono-devel`. I didn't pay close attention if it was updating or installing, but Mono should've been installed since I had SMAPI 2.5.5 on the computer before. I guess I could've tried this before opening the issue, but I forgot I even needed Mono for SMAPI. In any case, thanks!

**Comment by Pathoschild** [comment]
Thanks for the update! That dependency isn't needed on Linux/Mac, so I'll keep this open to change SMAPI's installer to avoid the issue. A few other Linux players seem to be affected too.

**Comment by Pathoschild** [comment]
Fixed in `develop` for the upcoming SMAPI 2.6-beta.4.

**Comment by mbkv** [comment]
This issue had a regression in 2.9.3. Fixed it by doing `apt update && apt install mono-devel` just like Kevin suggested.

```
Unhandled Exception:
System.IO.FileNotFoundException: Could not load file or assembly or one of its dependencies.
File name: 'System.IO.Compression.FileSystem, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
```
